data/src/main/java/io/gnosis/data/backend/dto/GateTransaction.kt

```diff
@@ -50,7 +50,11 @@ sealed class TransactionInfoDto {
     ) : TransactionInfoDto()
 
     data class Creation(
-        override val type: GateTransactionType = GateTransactionType.Creation
+        override val type: GateTransactionType = GateTransactionType.Creation,
+        val creator: Solidity.Address,
+        val transactionHash: String,
+        val implementation: Solidity.Address,
```

Comment:
what is implementation? Master copy address?

Reply:
Yeah, it was part of the inclusive renaming. Anywhere you see `implementation`, as far as contracts are concerned, think of it as `masterCopy`